tests/unit/components/Camera/StandardCamera.test.js

```diff
@@ -0,0 +1,65 @@
+import { fireEvent, render, screen } from "@testing-library/react-native";
+import StandardCamera from "components/Camera/StandardCamera";
+import { ObsEditContext } from "providers/contexts";
+import React from "react";
+import { View } from "react-native";
+
+const mockedNavigate = jest.fn();
+
+jest.mock( "@react-navigation/native", () => {
+  const actualNav = jest.requireActual( "@react-navigation/native" );
+  return {
+    ...actualNav,
+    useNavigation: () => ( {
+      navigate: mockedNavigate
+    } ),
+    useRoute: () => ( {} )
+  };
+} );
+
+const mockValue = {
+  addCameraPhotosToCurrentObservation: jest.fn(),
+  allObsPhotoUris: [],
+  cameraPreviewUris: []
+};
+
+const mockView = <View />;
+jest.mock( "components/Camera/CameraView", () => ( {
+  __esModule: true,
+  default: ( ) => mockView
+} ) );
+
+jest.mock( "components/Camera/FadeInOutView", () => ( {
+  __esModule: true,
+  default: () => mockView
+} ) );
+
+jest.mock( "components/Camera/PhotoPreview", () => ( {
+  __esModule: true,
+  default: () => mockView
+} ) );
```

Comment:
Cool. Stuff. I wish there was a good way to do `jest.mock` in helper methods instead of only globally or in a test like this.